client/src/modules/stock/movements/templates/action.cell.html

```diff
@@ -14,6 +14,13 @@
       </a>
     </li>
 
+    <li>
+      <a data-method="view-patient" href
+        ui-sref="stockInventories({ filters : [{ key : 'period', value : 'allTime', includeEmptyLot: 0 }, { key : 'depot_uuid', value : row.entity.depot_uuid }, { key : 'inventory_uuid', value : row.entity.inventory_uuid }]})">
```

Comment:
Could you change:
```js
 { key : 'depot_uuid', value : row.entity.depot_uuid },
```
to:
```js
 { key : 'depot_uuid', value : row.entity.depot_uuid, displayValue: row.entity.depot_text },
```
This will make the filter show the depot name on the target page.

Reply:
Could you also change:
```js
{ key : 'inventory_uuid', value : row.entity.inventory_uuid }
```
to:
```js
{ key : 'inventory_uuid', value : row.entity.inventory_uuid, displayValue: row.entity.text }
```
This will show the inventory label in the filter.